code/datums/world_topic.dm

```diff
@@ -236,3 +236,25 @@
 		.["shuttle_timer"] = SSshuttle.emergency.timeLeft()
 		// Shuttle timer, in seconds
 
+//Status Cog Support Code
+/datum/world_topic/whois
+	keyword = "whoIs"
+
+/datum/world_topic/whois/Run(list/input)
+	. = list()
+	.["players"] = GLOB.clients
+
+	return list2params(.)
+
+/datum/world_topic/getadmins
+	keyword = "getAdmins"
+
+/datum/world_topic/getadmins/Run(list/input)
```

Comment:
This also doesn't include any admins with 'Stealth Mode' enabled, if you wanted that.

Reply:
This is very much intentional.

Reply:
Fair enough 👍